vita3k/config/src/config.cpp

```diff
@@ -252,8 +252,14 @@ void merge_configs(Config &lhs, const Config &rhs, const std::string &new_pref_p
         lhs.hardware_flip = rhs.hardware_flip;
     if (lhs.performance_overlay != rhs.performance_overlay && (!init || rhs.performance_overlay))
         lhs.performance_overlay = rhs.performance_overlay;
-    if (lhs.backend_renderer != rhs.backend_renderer && (!init || rhs.backend_renderer != std::string("OpenGL")))
-        lhs.backend_renderer = rhs.backend_renderer;
+    if (lhs.backend_renderer != rhs.backend_renderer) {
+#ifdef USE_VULKAN
+        if (!init || rhs.backend_renderer != std::string("OpenGL"))
+#else
+        if (!init || lhs.backend_renderer != std::string("OpenGL"))
+#endif
+            lhs.backend_renderer = rhs.backend_renderer;
```

Comment:
Why is this different. This whole thing should be wrapped in a USE_VULKAN statement I think. It's not clear why changing `lhs` to `rhs` in a vulkan build will help accomplish the goal.

Reply:
for here is very special, my test have shown, it return all time on ogl, so i have founded special things can get works